schemas/2015-08-01/Microsoft.Cache.json

```diff
@@ -1,173 +1,210 @@
 {
-  "id": "https://schema.management.azure.com/schemas/2015-08-01/Microsoft.Cache.json",
+  "id": "https://schema.management.azure.com/schemas/2015-08-01/Microsoft.Cache.json#",
```

Comment:
While I would hope adding the '#' to schema id is a non-breaking change, I am not really sure, is that a question you've already considered?

Reply:
As part of the CIT test suite, we compile the root schemas using AJV, which validates the document plus all linked documents according to the json schema 4 spec. It'll complain if it can't resolve a reference, or if the id in the target document doesn't match the expected (e.g. if the date was incorrect). It's not complaining here, plus the majority of our schemas in this repo have been formatted with the trailing `#`, so I feel confident in saying this isn't going to be a breaking change. 

I've not been able to find much of an explanation in the [json schema 4 RFC](https://tools.ietf.org/html/draft-zyp-json-schema-04), but worth noting that all of their examples include the trailing `#`.